samples/azure-service-bus-netstandard/lock-renewal/sample.md

```diff
@@ -17,40 +17,40 @@ include: asb-connectionstring-xplat
+The Azure Service Bus transport sets `LockDuration` to 5 minutes by default, so the default `LockDuration` for the feature has the same value. `ExecuteRenewalBefore` is a `TimeSpan` specifying how soon to attempt lock renewal before the lock expires. The default is 10 seconds. Both settings may be overridden using the `EndpointConfiguration` API.
 
 snippet: override-lock-renewal-configuration
 
-In this sample, the `LockDuration` of the queue is modified to be 30 seconds, and lock renewal will take place 5 seconds before the lock expires.
+In the sample, `LockDuration` is set to 30 seconds, and `ExecuteRenewalBefore` is set to 5 seconds.
 
 ### Lock renewal behavior
 
-The `LockRenewalFeature` uses the two settings and registers the pipeline behavior `LockRenewalBehavior`, providing it with the `TimeSpan` to use for the lock renewal. With the endpoint configured for `LockDuration` of 30 seconds and renewal of 5 seconds before the lock expires, the lock token will be renewed every (`LockDuration` - `ExecuteRenewalBefore`) or 25 seconds.
+The `LockRenewalFeature` uses the two settings to register the `LockRenewalBehavior` pipeline behavior. With `LockDuration` set to 30 seconds and `ExecuteRenewalBefore` set to 5 seconds, the lock will be renewed every 25 seconds (`LockDuration` - `ExecuteRenewalBefore`).
 
-This behavior will process every incoming message going through the pipeline. Using [native message access](/transports/azure-service-bus/native-message-access.md), the behavior can get access to the message's lock token, required for lock renewal.
+The behavior processes every incoming message and uses [native message access](/transports/azure-service-bus/native-message-access.md) to get the message's lock token, which is required for lock renewal.
 
 snippet: native-message-access
 
-The call to extend the lock must be using the same Azure Service Bus connection object and the queue path used to receive the incoming message. The transport exposes this information via `TransportTransaction` and can be accessed in the following manner:
+The request to renew the lock must use the same Azure Service Bus connection object and queue path used to receive the incoming message. These items are available in the `TransportTransaction`:
 
 snippet: get-connection-and-path
 
-With connection object and queue path, an Azure ServiceBus can be created to renew the lock using the message's lock token obtained earlier. This is done in a background task, running in an infinite loop until the cancellation token passed in is signaled as canceled.
+With the lock token, connection object, and queue path, an Azure ServiceBus may be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signaled as canceled.
```

Comment:
What is "an Azure ServiceBus" in this context? Will the reader understand this?

Reply:
No, that's off. What it should be is
```suggestion
With the lock token, connection object, and queue path, an Azure Service Bus `MessageReceiver` object can be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signalled as cancelled.
```

Reply:
👍 btw the US English forms are "signaled" and "canceled".

```suggestion
With the lock token, connection object, and queue path, an Azure Service Bus `MessageReceiver` object may be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signaled as canceled.
```